spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindArgumentAssertions.java

```diff
@@ -21,24 +38,16 @@ public FindArgumentAssertions(BugReporter bugReporter) {
         super(bugReporter);
     }
 
-    /**
-     * Only interested in public methods
-     */
-    @Override
-    public void visit(Method obj) {
```

Comment:
Unlike the `visitClassContext(ClassContext)` function - which is inherited from the [ByteCodeScanningDetector](https://github.com/spotbugs/spotbugs/blob/master/spotbugs/src/main/java/edu/umd/cs/findbugs/BytecodeScanningDetector.java#L36) -, the `visit(Method)` function is not used for directing the run of the `sawOpcode` function, but to write the main logic of the detection.
The function is inherited from [BetterVisitor](https://github.com/spotbugs/spotbugs/blob/master/spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/BetterVisitor.java#L91) class and the function body is empty, so there is no need to call it.
Instead of this, I added the necessary check to the detect() method.